nucypher/blockchain/eth/actors.py

```diff
@@ -288,14 +291,42 @@ class Operator(BaseActor):
     class OperatorError(BaseActor.ActorError):
         pass
 
-    def __init__(self,
-                 is_me: bool,
-                 payment_method: ContractPayment,
-                 work_tracker: WorkTracker = None,
-                 operator_address: ChecksumAddress = None,
-                 *args, **kwargs):
+    def __init__(
+        self,
+        is_me: bool,
+        eth_provider_uri: str,
+        payment_method: ContractPayment,
+        work_tracker: Optional[WorkTracker] = None,
+        operator_address: Optional[ChecksumAddress] = None,
+        signer: Signer = None,
```

Comment:
Looking at the complete PR diff in order, this is the first line that made me go, "huh, I wonder why we're going in this direction."